api_spec/edge_api.json

```diff
@@ -178,6 +223,27 @@
                         "type": "string"
                     }
                 }
+            },
+            "BulkIdentitiesData": {
+                "required": true,
+                "default": {
+                    "name": "identifier"
+                },
+                "type": "object",
+                "properties": {
+                    "data": {
+                        "$ref": "#/components/schemas/IdentitiesRequestData"
+                    }
+                }
+            },
+            "Sample": {
```

Comment:
This needs a better name: 'Sample' is too generic. I would also say it's not a 'schema'. Can we create another key in the `components` object? So, this would end up at something like: `#/components/examples/PostIdentitiesExample`